### DISCUSSION TITLE
Windows Installer - list of installed files

### DISCUSSION THREAD

**Discussion by mynumu** [oobabooga/text-generation-webui#404] [General]
Hi all - anyone have a list of all the files and packages that get installed with the one click windows installer. Trying to remove them as have used the WSL version instead.

I see the text-generation-webui but the timestamps look like it touched a lot of the system32 installer files? mamba? anaconda?

Standard CLI for anaconda not working...

Thanks

**Comment by jepjoo** [comment]
Did you find an answer to this? Would like to know aswell.

**Reply by mynumu** [reply]
Hello..I didn’t and I ended up having to do a system restore. The install touched a lot of files in system32. My bad…should know not to install experimental stuff on my windows laptop by now ha ha…Im sure there’s no bad code in there but not easy to undo or remove…i think it may show you all the files on install if you capture the installer output. I’m using Virtual Box running Ubuntu instead on windows. The WSL was also a pain to get working.
